authentication/forms.py

```diff
@@ -20,18 +28,13 @@ class Meta:
         fields = ('username', 'password')
 
 
-class UpdateProfileForm(forms.ModelForm):
+class UpdateProfileForm(EmailDuplicationMixin, forms.ModelForm):
 
     class Meta:
         model = User
         fields = ('first_name', 'last_name', 'username', 'email')
 
-
-
-
-
-
-
-
-
-
+    def save(self):
```

Comment:
why we need to override `save` method of form here?

Reply:
Because when we save the form via form.save() method it saves the data into database but didn't hash the password field. So I override it in SignUpForm and make password hash manually. 

- Mistakenly I override into UpdateProfileForm also, now I removed it.